src/components/utility/Typography/Typography.module.css

```diff
@@ -0,0 +1,57 @@
+@import url('https://fonts.googleapis.com/css2?family=Livvic:ital,wght@0,600;0,700;1,600;1,700&display=swap');
```

Comment:
I have merged the PR, so you can remove this import from here, else it looks good

Reply:
You know, it occurs to me now that Typography will need to be global too. I'll leave the import and body font declaration in App.css for now, but it's something we might do later?